leakcanary-android-core/src/main/java/leakcanary/internal/utils/Size.kt

```diff
@@ -0,0 +1,14 @@
+package leakcanary.internal.utils
+
+
+// https://stackoverflow.com/a/3758880
+fun humanReadableByteCount(
+        bytes: Long,
+        si: Boolean
+): String {
+    val unit = if (si) 1000 else 1024
+    if (bytes < unit) return "$bytes B"
+    val exp = (Math.log(bytes.toDouble()) / Math.log(unit.toDouble())).toInt()
+    val pre = (if (si) "kMGTPE" else "KMGTPE")[exp - 1] + if (si) "" else "i"
+    return String.format("%.1f %sB", bytes / Math.pow(unit.toDouble(), exp.toDouble()), pre)
```

Comment:
Here, I kept original Stackoverflow impl not applying Kotlin linter suggestion (replacing Java Math APIs with Kotlin equivalents).
Shouldn't we fully embrace Kotlin here given that original source is already altered from Java to Kotlin?

Reply:
Yep, seems sensible to apply the Kotlin linter suggestion.

Reply:
I'm not sure to understand your answer, do you want me to make such impl more Kotlin idiomatic?

Reply:
I considered yes, I've added a simple commit to apply Lint suggestion. Can be easily removed if I misunderstood.

Reply:
yep, that's what I meant.